packages/manager/modules/pci/src/projects/project/edit/edit.routing.js

```diff
@@ -1,14 +1,104 @@
+import { Environment } from '@ovh-ux/manager-config';
+import get from 'lodash/get';
 import { MESSAGES_CONTAINER_NAME } from './edit.constant';
+import { PCI_HDS_ADDON } from '../project.constants';
 
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.edit', {
-    url: '/edit',
+    url: '/edit?cartId',
     views: {
       contentTab: 'pciProjectEdit',
     },
+    params: {
+      cartId: null,
+    },
     resolve: {
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('pci_projects_project_parameters'),
+
+      cart: /* @ngInject */ (
+        $stateParams,
+        prepareCart,
+        project,
+        orderCart,
+        hds,
+      ) => {
+        if (hds.option && !hds.isCertifiedProject && hds.isValidSupportLevel) {
+          return $stateParams.cartId
```

Comment:
`$stateParams` is deprecated : https://ui-router.github.io/ng1/docs/latest/modules/injectables.html#_stateparams

Reply:
Updated.
Thanks.